### PR TITLE
Add matview min adhoc ddl tests

### PR DESCRIPTION
**Pull request by jxchinaren**

### CODE REVIEW

File: tests/frontend/org/voltdb/TestAdhocAlterTable.java [hunk 1]
```diff
@@ -1277,4 +1277,143 @@ private void checkAlterTableSucceed(String sql) throws Exception {
...
+            // DROP a column in the view failed (the first column dropped)
+            //
+            checkAlterTableFailed("alter table FOO drop column VAL;", "column is referenced in");
+            assertTrue(doesColumnExist("FOO", "VAL"));
+
+            //
+            // DROP a column not in the view failed (the first column dropped)
+            //
+            // -- JSON plans of the view query will change
+            checkAlterTableSucceed("alter table FOO drop column NUM1;");
```
Comment (paulmartel) [inline tests/frontend/org/voltdb/TestAdhocAlterTable.java]: Comment says "failed" -- test says "succeeded".

Comment (jxchinaren) [inline tests/frontend/org/voltdb/TestAdhocAlterTable.java]: OK, I need to change the original tests also. Bad copy.

File: tests/frontend/org/voltdb/TestAdhocAlterTable.java [hunk 2]
```diff
@@ -1277,4 +1277,143 @@ private void checkAlterTableSucceed(String sql) throws Exception {
...
+            assertFalse(doesColumnExist("FOO", "NUM1"));
+
+            //
+            // DROP a column not in the view failed (the first column dropped)
+            //
+            // -- JSON plans of the view query will change
+            checkAlterTableFailed("alter table BAR drop column NUM1;",
+                    "May not dynamically modify field 'predicate' of schema object 'MaterializedViewInfo{BARVIEW}'");
+            assertTrue(doesColumnExist("BAR", "NUM1"));
```
Comment (paulmartel) [inline tests/frontend/org/voltdb/TestAdhocAlterTable.java]: The way this test passes for BAR and fails for FOO makes me want to test more cases with both BAR and FOO -- not just with FOO. Maybe there are other differences.

**Comment by paulmartel** [comment]
Looks good.

**Pull request merged**